### PR TITLE
Add 6 OP scripts for 2.0 version

### PR DESCRIPTION
**Pull request by AshburnLee**
This PR added 5 OP scripts: 

1. nornalize
2. switch_case
3. sigmoid_cross_entropy_with_logits
4. batch_norm
5. increment

The results of the above 5 scripts are as expected. 'mean' has been deleted temporarily

### CODE REVIEW

File: api/tests_v2/batch_norm.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            running_mean=running_mean,
+            running_var=running_var,
+            weight=weight,
+            bias=bias,
+            epsilon=config.epsilon,
+            momentum=config.momentum,
+            training=False,
+            data_format=config.data_layout)
+
+        self.feed_vars = [input, running_mean, running_var, weight, bias]
```
Comment (Xreki) [inline api/tests_v2/batch_norm.py:78]: feed变量数，paddle和tf应该保持一致

File: api/tests_v2/increment.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+class PDIncrement(PaddleAPIBenchmarkBase):
+    def build_program(self, config):
+        x = self.variable(name='x', shape=config.x_shape, dtype=config.x_dtype)
+        x.stop_gradient = False
+        result = paddle.increment(x, value=config.value)
+
+        self.feed_vars = [x]
+        self.fetch_vars = [result]
+        if config.backward:
```
Comment (Xreki) [inline api/tests_v2/increment.py:26]: increment没有反向

File: api/tests_v2/mean.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+class MeanConfig(APIConfig):
+    def __init__(self):
+        super(MeanConfig, self).__init__('mean')
+
+
+class PDMean(PaddleAPIBenchmarkBase):
+    def build_program(self, config):
+        x = self.variable(name='x', shape=config.x_shape, dtype=config.x_dtype)
+        result = paddle.mean(x=x)
```
Comment (Xreki) [inline api/tests_v2/mean.py:26]: reduce里面包括paddle.mean了

Comment (GaoWei8) [reply api/tests_v2/mean.py:26]: 这个是调用原来的mean接口吗？reduce_mean确实也修改为mean接口。但是两个接口的参数不同。

File: api/tests_v2/increment.py (hunk 2 above)
Comment (AshburnLee) [reply api/tests_v2/increment.py:26]: Done，increment通过

File: api/tests_v2/batch_norm.py [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+        self.fetch_vars = [result]
+        if config.backward:
+            self.append_gradients(
+                result, [input, running_mean, running_var, weight, bias])
```
[the remark this answers is not in the document]
Comment (AshburnLee) [reply api/tests_v2/batch_norm.py]: DONE

File: api/tests_v2/batch_norm.py (hunk 1 above)
Comment (AshburnLee) [reply api/tests_v2/batch_norm.py]: DONE

**Review by Xreki — APPROVED** [review]

**Review by Xreki — APPROVED** [review]

**Pull request merged**